pkg/cli/cmd/radinit/recipe.go

```diff
@@ -28,14 +28,32 @@ import (
 	msg_ctrl "github.com/radius-project/radius/pkg/messagingrp/frontend/controller"
 	recipe_types "github.com/radius-project/radius/pkg/recipes"
 	"github.com/radius-project/radius/pkg/to"
+	"github.com/radius-project/radius/pkg/ucp/ucplog"
 	"github.com/radius-project/radius/pkg/version"
 	"oras.land/oras-go/v2/registry/remote"
 )
 
 const (
-	DevRecipesRegistry = "ghcr.io/radius-project"
+	// RecipeRepositoryPrefix is the prefix for the repository path.
+	RecipeRepositoryPrefix = "ghcr.io/radius-project/recipes/local-dev/"
 )
 
+// availableDevRecipes returns the list of available dev recipes.
+//
+// If we want to add a new recipe, we need to add it here.
+func availableDevRecipes() []string {
```

Comment:
We can add all the resources we have here so that we don't need to do another release if we'd like to add a new dev recipe. Adding the recipe to the registry will be enough then.